### PR TITLE
fix paths in RubyMine-EAP zap

### PR DESCRIPTION
**Pull request by yurikoles**
Sorry, I will fix commit soon.

### CODE REVIEW

**Comment by fanquake** [comment]
@yurikoles Is the commit fixed?

**Comment by yurikoles** [comment]
@fanquake,
I had forgot to run `brew up` before previous commit and there was merge conflict.

**Comment by yurikoles** [comment]
The paths in stable Cask zap is also outdated (e.g. '70' vs '80'), but I don't want spent time on this issue since new stable release with new version scheme is expected soon.

@fanquake could you advice me better way of version extraction than I had used in this cask?

File: Casks/rubymine-eap.rb [hunk 1]
```diff
@@ -10,11 +10,12 @@
...
 
   zap delete: [
-                '~/Library/Preferences/com.jetbrains.rubymine-EAP.plist',
-                '~/Library/Preferences/RubyMine80',
-                '~/Library/Application Support/RubyMine-EAP',
-                '~/Library/Caches/RubyMine70',
-                '~/Library/Logs/RubyMine70',
+
+                '~/Library/Preferences/com.jetbrains.rubymine.plist',
+                "~/Library/Preferences/RubyMine#{version.split('-')[0]}",
```
Comment (vitorgalvao) [inline Casks/rubymine-eap.rb]: `#{version.major_minor}` should do it.

File: Casks/rubymine-eap.rb [hunk 2]
```diff
@@ -10,11 +10,12 @@
   app 'RubyMine.app'
 
   zap delete: [
-                '~/Library/Preferences/com.jetbrains.rubymine-EAP.plist',
-                '~/Library/Preferences/RubyMine80',
-                '~/Library/Application Support/RubyMine-EAP',
-                '~/Library/Caches/RubyMine70',
-                '~/Library/Logs/RubyMine70',
+
```
Comment (vitorgalvao) [inline Casks/rubymine-eap.rb]: Empty line must go.

Comment (yurikoles) [inline Casks/rubymine-eap.rb]:
```
== rubymine-eap.rb ==
C: 21:  4: [Corrected] Final newline missing.

1 file inspected, 1 offense detected, 1 offense corrected
```

Comment (vitorgalvao) [inline Casks/rubymine-eap.rb]: This is not a final newline, it’s an empty line that shouldn’t be there.

Comment (yurikoles) [inline Casks/rubymine-eap.rb]: sorry, done

**Comment by yurikoles** [comment]
@vitorgalvao thanks for corrections, all is fixed, please merge

**Comment by vitorgalvao** [comment]
> Is there any API docs for cask?

[Docs](https://github.com/caskroom/homebrew-cask/tree/master/doc).

**Comment by vitorgalvao** [comment]
Closing in favour of https://github.com/caskroom/homebrew-versions/pull/1873/files.

**Pull request closed without merging**